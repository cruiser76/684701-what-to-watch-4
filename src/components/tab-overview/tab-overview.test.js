import React from 'react';
import renderer from 'react-test-renderer';

import TabOverview from './tab-overview.jsx';

const movie = {
  img: {
    src: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    posterSrc: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    bgSrc: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`
  },
  brief: {
    title: `Fantastic Beasts: The Crimes of Grindelwald`,
    genre: `Drama`,
    year: `2000`,
    score: `9`,
    level: `Good`,
    starring: [`Ivan`]
  },
  link: `movie-page.html`,
  key: `1`,
};

const props = {
  movie,
  activeElement: `Overview`,
  loadReviews: () => {},
};

it(`Tab should render correctly`, () => {
  const tree = renderer
    .create(
        <TabOverview
          {...props}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
