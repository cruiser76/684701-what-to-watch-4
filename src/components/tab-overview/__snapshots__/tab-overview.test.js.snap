// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab should render correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      9
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Good
      </span>
      <span
        className="movie-rating__count"
      >
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p />
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Ivan
      </strong>
    </p>
  </div>,
]
`;
